frontend/src/components/UserlocationCard.tsx

```diff
@@ -0,0 +1,58 @@
+import {Userlocation} from "./Entities.ts";
+
+type UserlocationCardProps = {
+    userlocation: Userlocation
+}
+
+export default function UserlocationCard (props: UserlocationCardProps) {
+
+    return (
+        <div id="edit-container">
+
+            These are the credentials of your individual location.
+            Based on the information you're providing below, we are aiming to recommend the perfect groupset for your road bike.
```

Comment:
```suggestion
            <p>These are the credentials of your individual location.
            Based on the information you're providing below, we are aiming to recommend the perfect groupset for your road bike.</p>
```
Man sollte Text ohne tag nicht schreiben.